README.md

```diff
@@ -124,10 +147,10 @@ bool presence = isUserInList(bytes32 fractaId, string listId);
 
 Every `fractalId` in the DID Registry corresponds to a unique human. Use cases requiring additional guarantees, such as KYC/AML, can also make use of the following lists.
 
-| `listId` | Meaning |
-| :--- | :--- |
-| `basic` | Passed KYC level _basic_ |
-| `plus` | Passed KYC level _plus_ |
+| `listId`       | Meaning                                                                                                                                                                 |
+| :------------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `basic`        | Passed KYC level _basic_                                                                                                                                                |
+| `plus`         | Passed KYC level _plus_                                                                                                                                                 |
 | `residency_xy` | Resident in country _xy_ ([ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) country codes).<br>E.g. `residency_ca`, `residency_de`, `residency_us` |
```

Comment:
these changes were simply the default code formatter not liking your Markdown 😅
we did not make any changes to the Registry — was this a derp or are you suggesting we add it because we're already supporting it in the credentials and could do the same for the registry in the future?

Reply:
The registry supports arbitrary lists, and while citizend's doesn't require it, other deployments do, so we will create those lists too.